lib/ansible/modules/windows/win_psmodule.ps1

```diff
@@ -117,15 +118,27 @@ Function Install-PsModule {
     else {      
       try{
         # Install NuGet Provider if needed
-        Install-NugetProvider -CheckMode $CheckMode
-
-        # Check Powershell Version (-AllowClobber was introduced in early version only)
-        if ($PsVersion.Minor -ge 1){
-          Install-Module -Name $Name -Force -ErrorAction Stop -Whatif:$CheckMode -AllowClobber:$AllowClobber | out-null
+        Install-NugetProvider -CheckMode $CheckMode;
+
+        $ht = @{
+            Name      = $Name;
+            WhatIf    = $CheckMode;
+            ErrorAction = "Stop";
+            Force     = $true;
+        };
+
+        # If specified, use repository name to select module source
+        if ($Repository) {
+            $ht["Repository"] = "$Repository";
```

Comment:
Can just do `$ht["Repository"] = $Repository`, no need to enclose it in a string. Another part which is just a minor nitpick and you don't have to do anything is that lines don't need to end with `;` in PowerShell. Personally I think it detracts from the code but that's just me.

Reply:
You're right, it's just a preference on my part.
Also, double-quoting forces string conversion, which has become a habit of mine after being burned with some weird scripts, but here it's not necessary. I guess it just helps the eye pick up the type of the data from the context.

Reply:
Thank you for your continued help and advices, without your aid I would have had a hard time figuring out how things clicked!

It was my pleasure, and feel good to put my work to the use of so many others.